pytest.ini

```diff
@@ -2,6 +2,7 @@
 env =
   FLASK_ENV=unit_test
   D:E2E_NOTIFY_FIND_API_KEY=set-me-in-your-environment
+  D:E2E_DEVTEST_BASIC_AUTH=set-me-in-your-environment
```

Comment:
Could we set these in the `unit_test.py` config rather than here? What's the trade-offs there.

Reply:
Hummm yes we can. I suppose I thought that the `config` module was more specifically about _app_ config, and this blurs that line a little .. but maybe that's not a super-important distinction.

Moved in 44d0d25b

Reply:
Yeah, I see what you mean, although I don't think we'd really run `unit_test` config in any other context.